To avoid undefined IndexFastScan.search_dispatch_implem's symbol errors, define template. 

Copyright (c) 2024 Di-Is

This software is released under the MIT License.
http://opensource.org/licenses/mit-license.php

diff --git a/faiss/faiss/IndexFastScan.cpp b/faiss/faiss/IndexFastScan.cpp
index 2dfb2f55..561964c9 100644
--- a/faiss/faiss/IndexFastScan.cpp
+++ b/faiss/faiss/IndexFastScan.cpp
@@ -341,6 +341,9 @@ void IndexFastScan::search_dispatch_implem(
         FAISS_THROW_FMT("invalid implem %d impl=%d", implem, impl);
     }
 }
+// Fix undefined symbol using static faiss library in release build  
+template void IndexFastScan::search_dispatch_implem<true>(idx_t, const float*, idx_t, float*, idx_t*, const NormTableScaler*) const;
+template void IndexFastScan::search_dispatch_implem<false>(idx_t, const float*, idx_t, float*, idx_t*, const NormTableScaler*) const;
 
 template <class Cfloat>
 void IndexFastScan::search_implem_234(
